src/models/backbones/resnet.py

```diff
@@ -143,17 +139,25 @@ def forward(self, x: torch.Tensor):
 
         return x
 
-    def get_features_info(self, output_channels: List[int], module_names: List[int], strides: List[int]):
-        """See documentation in constructor."""
-        features_info = []
-        for num_channels, module_name, stride in zip(output_channels, module_names, strides):
-            feature_info = FeatureInfo(module_name=module_name, num_channels=num_channels, stride=stride)
-            features_info.append(feature_info)
-        return features_info
-
-    def get_forward_output_channels(self) -> Union[int, List[int]]:
-        """Return number of output channels."""
-        return self.num_features
+    def forward_features(self, x: torch.Tensor):
+        """Forward method."""
+        features = [x]
+        x = self.convbnact(x)
+        x = self.maxpool(x)
```

Comment:
Shouldn't we also return the feature map after the "stem" layer?

Reply:
In previous version of TorchOk it not returned, but i think we should to give flexibility